arch/cortex-m/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+// Copyright Tock Contributors
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment:
Should there be parentheses around these? The SPDX website does not show them in its examples except in cases of ambiguity. https://spdx.dev/ids/